### PR TITLE
mdless 1.0.32 (new formula)

### PR DESCRIPTION
**Pull request by bugfest**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/mdless.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  def install
+    ENV["GEM_HOME"] = libexec
+    system "gem", "build", "#{name}.gemspec"
+    system "gem", "install", "#{name}-#{version}.gem"
+    bin.install libexec/"bin/#{name}"
+    bin.env_script_all_files(libexec/"bin", GEM_HOME: ENV["GEM_HOME"])
+  end
+
+  test do
+    assert_match "mdless #{version}", shell_output("#{bin}/mdless --version")
```
[the remark this answers is not in the document]
Comment (bugfest) [reply Formula/mdless.rb:19]: I added some tests to check whether `mdless` properly parses markdown and outputs expected strings

File: Formula/mdless.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      ## title second level
+    EOS
+    shell_output("#{bin}/mdless --no-color -P test.md").chomp.split("\n").each do |line|
+      if line.start_with?("title first level")
+        assert line.end_with?("==="), "first level title expected to use === delimiter"
+      end
+      if line.start_with?("title second level")
+        assert line.end_with?("---"), "second level title expected to use --- delimiter"
+      end
+    end
```
Comment (SMillerDev) [inline Formula/mdless.rb:31]:
```suggestion
    out =  shell_output("#{bin}/mdless --no-color -P test.md")
    assert_match /^title first level .* ===$/, out
    assert_match /^title second level .* ---$/, out
```

Comment (bugfest) [reply Formula/mdless.rb:31]: Ah! I wasn't aware `assert_match` supported multi-line matching. Thanks for the suggestion @SMillerDev. Gonna adapt it a bit and update the block

Comment (bugfest) [reply Formula/mdless.rb]: Done! ([d93699d](https://github.com/Homebrew/homebrew-core/pull/121410/commits/d93699d54b8d3001b10155dcf913d6a6833b065c))

Comment (bugfest) [reply Formula/mdless.rb]: Completed by [2540f94](https://github.com/Homebrew/homebrew-core/pull/121410/commits/2540f94d8757d3e750fff9942d09cdb3bd3ae6d0)

**Review by SMillerDev — APPROVED** [review]
Thanks @bugfest  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**